### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by apu52**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs Update
- **Why was this change needed?** (You can also link to an open issue here)
Hello Sir/ma'am,
I have updated readme.md file
- **Other information**:
I have made some changes in the readme file
- I have updated some alignments
- Made some Bold font to some alphabets etc.

### CODE REVIEW

**Comment by apu52** [comment]
Hey @dartpain, could you take a look at the changes and merge the PR if everything looks good? Thanks!

**Comment by apu52** [comment]
Hey @dartpain, could you take a look at the changes and merge the PR if everything looks good? Thanks!

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Otherwise, refer to this Guide:
 3. Run [./run-with-docker-compose.sh](https://github.com/arc53/DocsGPT/blob/main/run-with-docker-compose.sh).
 4. Navigate to http://localhost:5173/.
 
-To stop, just run `Ctrl + C`.
+5. To stop, just run `Ctrl + C`.
```
Comment (dartpain) [inline README.md:103]: No need to have 5. here

**Review by dartpain — CHANGES_REQUESTED** [review]
Just a small change

**Pull request closed without merging**